secrets/eap-app-secret.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
If you'd like to switch to yaml, I think some of the newer images have been doing that.  It's certainly easier to read and maintain, and is preferred by the openshift team.

Reply:
Same comment applies to all the json files.